.github/workflows/daily-test.yml

```diff
@@ -0,0 +1,30 @@
+name: Daily Test
+
+on:
+  workflow_dispatch:
+  schedule:
+    # 8am daily Mon-Fri
+    - cron: '0 8 * * 1-5'
+
+jobs:
+  run_tests:
+    uses: ./.github/workflows/test.yml
+    with:
+      reason: daily-build
+  failure:
+    needs:
+      - run_tests
+    if: ${{ failure() }}
+    name: Communicate failure
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set current job url in SLACK_FOOTER env var
+        run: echo "SLACK_FOOTER=<$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID|Click here to go to the job logs>" >> $GITHUB_ENV
+
+      - name: Slack Notification on Failure
+        uses: rtCamp/action-slack-notify@v2.2.0
+        env:
+          SLACK_TITLE: 'prisma/prisma daily build failed :x:'
+          SLACK_COLOR: '#FF0000'
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+          SLACK_CHANNEL: tmp-daily-build-notification-tests
```

Comment:
Usually we name the channel like `feed-` so maybe `feed-prisma-daily-build-failures`

Reply:
Yea, I'd move it to a proper channel once I'll be sure it is working, just did not want to spam actual channels meanwhile

Reply:
I think you can change this already now to avoid the later PR - the channel will be a new one anyway.